src/views/Manager/Navbar.vue

```diff
@@ -19,17 +19,32 @@
           >
           </Button>
         </router-link>
-        <router-link v-if="token" to="/admin/products" class="link-content">
-          <Button icon="pi pi-fw pi-user" class="p-button-text p-button-plain">
-          </Button>
-        </router-link>
+
+        <Button
+          v-if="token"
```

Comment:
這樣讀起來比較直覺一點

```suggestion
          v-if="logged_in"
```

logged_in() {
  return Cookies.get("lemonToken") != undefined
}